src/components/home/header.tsx

```diff
@@ -0,0 +1,58 @@
+export default function Header() {
+  return (
+    <header
+      className="bg-white py-4 mx-0 md:mx-16"
+      style={{
+        fontFamily:
+          'Lexend, Manrope, Poppins, system-ui, -apple-system, Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, Liberation Sans, sans-serif',
+      }}
+    >
+      <div className="mx-auto p-4 flex flex-col md:flex-row justify-between gap-4 md:items-center">
+        <div className="flex items-center gap-2">
+          <img src="/src/assets/icons/icon1.svg" alt="Shipping Icon" />
+          <div className="flex flex-col gap-2">
```

Comment:
make it a reusable component, and I would suggest to make the List of objects that include image, title and description and loop through it.

Reply:
done